init-tools.cmd

```diff
@@ -34,21 +29,15 @@ if exist "%BUILD_TOOLS_SEMAPHORE%" (
 
 if exist "%TOOLRUNTIME_DIR%" rmdir /S /Q "%TOOLRUNTIME_DIR%"
 
-if NOT exist "%PROJECT_JSON_PATH%" mkdir "%PROJECT_JSON_PATH%"
-echo %PROJECT_JSON_CONTENTS% > "%PROJECT_JSON_FILE%"
 echo Running %0 > "%INIT_TOOLS_LOG%"
 
 set /p DOTNET_VERSION=< "%~dp0DotnetCLIVersion.txt"
-if exist "%DOTNET_CMD%" goto :afterdotnetinstall
-
-:: core-setup repo has a global.json which causes a conflict with the init-tools publish step
-if NOT exist "%PACKAGES_DIR%" mkdir %PACKAGES_DIR% > nul 2>&1
-echo { "projects": [] } > %DUMMY_GLOBAL_JSON_PATH%
+if exist "%DOTNET_CMD%" goto :afterdotnetrestore
 
 echo Installing dotnet cli...
 if NOT exist "%DOTNET_PATH%" mkdir "%DOTNET_PATH%"
 set DOTNET_ZIP_NAME=dotnet-dev-win-x64.%DOTNET_VERSION%.zip
-set DOTNET_REMOTE_PATH=https://dotnetcli.azureedge.net/dotnet/preview/Binaries/%DOTNET_VERSION%/%DOTNET_ZIP_NAME%
+set DOTNET_REMOTE_PATH=https://dotnetcli.blob.core.windows.net/dotnet/Sdk/%DOTNET_VERSION%/%DOTNET_ZIP_NAME%
```

Comment:
I specifically changed this in core-setup to use `azureedge`.  We need to use that URL because it is a CDN that is cached.  Thus, if you don't live on the west coast of the US, your download times speed up greatly when using `azureedge`.

Yes, this should propagate to all other repos who are doing it wrong.

In all honesty, this code (and all other repos) should be using the `dotnet-install` script, which already uses `azureedge`.